servers/quarkus-tests/src/main/java/org/projectnessie/quarkus/tests/profiles/KeycloakTestResourceLifecycleManager.java

```diff
@@ -205,14 +212,25 @@ private void configureRealm(RealmRepresentation realm) {
     realm.getRoles().getRealm().add(new RoleRepresentation("admin", null, false));
     realm.getRoles().getRealm().add(new RoleRepresentation("confidential", null, false));
 
-    realm.getClients().add(CustomKeycloakContainer.createServiceClient(serviceClientId));
+    realm
+        .getClients()
+        .add(CustomKeycloakContainer.createServiceClient(serviceClientId, clientScopes));
     realm.getClients().add(CustomKeycloakContainer.createWebAppClient(webAppClientId));
 
     realm.getUsers().add(CustomKeycloakContainer.createUser("alice", tokenUserRoles));
     realm.getUsers().add(CustomKeycloakContainer.createUser("admin", tokenAdminRoles));
-    realm
-        .getUsers()
-        .add(CustomKeycloakContainer.createUser("jdoe", Arrays.asList("user", "confidential")));
+
+    List<ClientScopeRepresentation> scopes =
+        clientScopes.stream()
+            .map(
+                scopeName -> {
+                  ClientScopeRepresentation scope = new ClientScopeRepresentation();
+                  scope.setName(scopeName);
+                  scope.setProtocol("openid-connect");
+                  return scope;
+                })
+            .collect(Collectors.toList());
```

Comment:
Nit: can use `.toList()` instead of `.collect(Collectors.toList())` here
```suggestion
            .toList();
```